### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by svenvg93**
## 📃 Description
Just a suggestion ;)
Looking at the amount issues that has been created due to people using the `latest `tag with auto updates. Pin the version in the README, and link the LSIO page to check for the last version number.,

### CODE REVIEW

**Comment by svenvg93** [comment]
If you agree i will as well submit a PR for the docs :)

**Comment by alexjustesen** [comment]
I'm thinking this is probably a good idea, now am I going to remember to do this during a release? 🤷‍♂️

File: README.md [hunk 1]
```diff
@@ -50,9 +50,12 @@ services:
         volumes:
             - /path/to/data:/config
             - /path/to-custom-ssl-keys:/config/keys
-        image: lscr.io/linuxserver/speedtest-tracker:latest
+        image: lscr.io/linuxserver/speedtest-tracker:0.20.5
         restart: unless-stopped
 ```
+## Image version
+
+Check [here](https://fleet.linuxserver.io/image?name=linuxserver/speedtest-tracker) to see what the latest version is. Due to breaking changes we don't advise to use the `latest` tag. Best pratice is to pin it to a version.
```
Comment (alexjustesen) [inline README.md:58]: Do you mind simplifying this to just

"A full list of released versions can be found here: lsio link"

**Review by alexjustesen — CHANGES_REQUESTED** [review]
Small change to simplify the note

File: README.md (hunk 1 above)
Comment (svenvg93) [reply README.md]: Done!

**Comment by svenvg93** [comment]
> I'm thinking this is probably a good idea, now am I going to remember to do this during a release? 🤷‍♂️

Maybe, maybe not hahaha

**Review by alexjustesen — APPROVED** [review]

**Pull request merged**